### PR TITLE
Use optional env var for show_progress in Ag3, Af1

### PR DESCRIPTION
**Pull request by leehart**
Re: issue #597 

I'm still wondering if we can put this logic in the common base class `AnophelesDataResource` instead.

### CODE REVIEW

File: malariagen_data/af1.py [hunk 1]
```diff
@@ -91,6 +92,18 @@ def __init__(
...
+            # Get the env var, if it exists.
+            show_progress_env = os.getenv("MGEN_SHOW_PROGRESS")
+
+            # If the env var does not exist, then use the class default.
+            # Otherwise, convert the env var value to a boolean and use that.
+            if show_progress_env is None:
+                show_progress = True
+            else:
+                show_progress = show_progress_env.lower() in ("true", "1", "yes", "on")
+
```
Comment (alimanfoo) [inline malariagen_data/af1.py:106]: The logic looks good but could move this up to `AnophelesBase.__init__`.

Comment (leehart) [reply malariagen_data/af1.py:106]: Confusingly, `AnophelesBase.__init__` currently defaults `show_progress` to `False`, whereas `Ag3` and `Af1` currently default to `True`. I expect this is just an historical artefact, so I'll set the default as `True`.

File: malariagen_data/ag3.py [hunk 2]
```diff
@@ -150,6 +150,8 @@ def __init__(
         tqdm_class=None,
         **storage_options,  # used by fsspec via init_filesystem()
     ):
+        print("Ag3 show_progress", show_progress)
+
```
Comment (alimanfoo) [inline malariagen_data/ag3.py:154]:
```suggestion
```

**Review by alimanfoo — COMMENTED** [review]
Looks good, just a print statement to remove.

**Review by alimanfoo — APPROVED** [review]
All good, thanks!

**Pull request merged**